bitcoin/psbt.c

```diff
@@ -616,9 +608,9 @@ bool psbt_finalize(struct wally_psbt *psbt)
 	for (size_t i = 0; i < psbt->num_inputs; i++) {
 		struct wally_psbt_input *input = &psbt->inputs[i];
 		struct wally_tx_witness_stack *stack;
-
-		if (!is_anchor_witness_script(input->witness_script,
-					      input->witness_script_len))
+		const struct wally_map_item *input_witness_script = wally_map_get_integer(&input->psbt_fields, /* PSBT_IN_WITNESS_SCRIPT */ 0x05);
```

Comment:
OpenBSD has a rule against using initializers in declarations.  I am sympathetic, but not dogmatic on this, but here I think it really deserves a separate line.

Reply:
But probably just call it `iws` given it's so hyperlocal...

Reply:
done